pkgs/tools/security/zeekscript/default.nix

```diff
@@ -36,5 +36,7 @@ python3.pkgs.buildPythonApplication rec {
     changelog = "https://github.com/zeek/zeekscript/blob/v${version}/CHANGES";
     license = licenses.bsd3;
     maintainers = with maintainers; [ fab tobim ];
+    # Incompatible with tree-sitter > 0.21.
+    broken = true;
```

Comment:
Perhaps an update to 1.2.9 could help.

https://github.com/zeek/zeekscript/blob/a35664f7f7c61f65093f11681b8b86911fd6ea9d/setup.py#L83

Reply:
It doesn't. The current development snapshot is compatible, but no release yet. And updating the package for that is essentially a rewrite.